rest/model/test.go

```diff
@@ -120,25 +99,7 @@ func (at *APITest) BuildFromService(st interface{}) error {
 }
 
 func (at *APITest) ToService() (interface{}, error) {
-	catcher := grip.NewBasicCatcher()
-	start, err := FromTimePtr(at.StartTime)
-	catcher.Add(err)
-	end, err := FromTimePtr(at.EndTime)
-	catcher.Add(err)
-	if catcher.HasErrors() {
-		return nil, catcher.Resolve()
-	}
-	return &testresult.TestResult{
-		Status:          utility.FromStringPtr(at.Status),
-		TestFile:        utility.FromStringPtr(at.TestFile),
-		DisplayTestName: utility.FromStringPtr(at.DisplayTestName),
-		URL:             utility.FromStringPtr(at.Logs.URL),
-		URLRaw:          utility.FromStringPtr(at.Logs.URLRaw),
-		LogID:           utility.FromStringPtr(at.Logs.LogId),
-		LineNum:         at.Logs.LineNum,
-		ExitCode:        at.ExitCode,
-		StartTime:       utility.ToPythonTime(start),
-		EndTime:         utility.ToPythonTime(end),
-		GroupID:         utility.FromStringPtr(at.GroupId),
-	}, nil
+	// It is not valid translate an APITest object to a TestResult object
+	// due to data loss.
+	return nil, errors.New("not implemented")
```

Comment:
If this isn't implemented, can we just remove `(*APITest).ToService` entirely? Most REST models don't have to satisfy the `rest.Model` interface, so I'm assuming this one doesn't have to either.